fiduswriter/document/static/js/modules/editor/index.js

```diff
@@ -387,6 +400,31 @@ export class Editor {
         })
     }
 
+
+    handleAccessRightModification() {
+        // This function when invoked creates a copy of document in FW format and closes editor operation.
+        new ExportFidusFile(
+            this.getDoc({'use_current_view': true}),
+            this.mod.db.bibDB,
+            this.mod.db.imageDB
+        )
+        const accessRightModifiedDialog = new Dialog({
+            title: gettext('Access rights modified'),
+            id: "access_rights_modified",
+            body: gettext('Your access rights were modified while you were offline so we cannot save your work to the server and we download it to your computer instead. Please consider importing it into a new document.'),
```

Comment:
How about:

Your access rights were modified while you were offline so we cannot save your work to the server any longer and **you need to** download it to your computer instead. Please consider importing it into a new document.

Reply:
I've updated the message with your suggestion. I've also updated the message that we show when user's session expired when he came back online.